VERSION

```diff
@@ -1 +1 @@
-3.1.0-rc2
+3.1.0
```

Comment:
There's a typo in the commit message and title, it says `3.0.0` instead of `3.1.0`

Reply:
Fixed, thanks!